electionleaflets/apps/leaflets/models.py

```diff
@@ -92,24 +93,62 @@ def save(self, *args, **kwargs):
 
 
 class LeafletImage(models.Model):
+    ORIENTATION_CHOICES = (
+        (1, 'Horizontal (normal)'),
+        (2, 'Mirror horizontal'),
+        (3, 'Rotate 180'),
+        (4, 'Mirror vertical'),
+        (5, 'Mirror horizontal and rotate 270 CW'),
+        (6, 'Rotate 90 CW'),
+        (7, 'Mirror horizontal and rotate 90 CW'),
+        (8, 'Rotate 270 CW'),
+    )
+
     leaflet = models.ForeignKey(Leaflet, related_name='images')
     image = ImageField(upload_to="leaflets", max_length=255)
     raw_image = ImageField(upload_to="raw_leaflets", blank=True, max_length=255)
     legacy_image_key = models.CharField(max_length=255, blank=True)
     image_type = models.CharField(choices=constants.IMAGE_TYPES,
         null=True, blank=True, max_length=255)
     image_text = models.TextField(blank=True)
+    orientation = models.PositiveSmallIntegerField(choices=ORIENTATION_CHOICES, default=1)
+    exif_data = models.BinaryField(blank=True)
 
     class Meta:
         ordering = ['image_type']
 
+    def _remove_exif_data(self):
+        full_image = Image.open(self.image)
+
+        if full_image.info.get('exif'):
+            exif_dict = piexif.load(full_image.info['exif'])
```

Comment:
Could you extract EXIF reading to some methods on this model? So we can access `self.exif_data` without having to import `pyexif` each time. I think that abstraction should be able to work for this data too?

Something like 

```python

def exif_data(self, data=None, key=None):
    if not data:
        data = self.exif_data
    [do stuff that returns the data or the value for the provided key]
```

Is this a silly idea? Just seems we're going to want to be able to read the exif data at a later date using something like this.